single_chart_migration/mongo.yaml

```diff
@@ -0,0 +1,251 @@
+# Source: px-central/templates/px-backup/pxcentral-mongodb.yaml
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: pxc-backup-mongodb-trail
+  labels:
+    app.kubernetes.io/component: pxc-backup-mongodb-trail
```

Comment:
We can remove these levels from each resources for trial run. At least the labels related to helm.

Reply:
Removed the labels.